frontend/src/components/immunohistochemistry/ImmunohistochemistryDashboard.js

```diff
@@ -284,6 +285,7 @@ function ImmunohistochemistryDashboard() {
     <>
       {notificationVisible === true ? <AlertDialog /> : ""}
       {loading && <Loading description="Loading Dasboard..." />}
+      <GenericHomeBreadCrumb/>
```

Comment:
Yes it's going to eventually this is not final, but you can still suggest change. I wanted this to work for validation because it has a nested breadcrumb hierarchy if it pulls up then this will go for others